tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2394 @@
+@charset UTF-8; /* Invalid, without ' or ", the charset is not a CSS <string> */
+
+`;
+
+exports[`counter-style.css 1`] = `
+@counter-style identifier {
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style identifier {
+system: cyclic;
+symbols: "\\1F44D";
+suffix: " ";
+}
+@counter-style identifier{
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style  identifier  {
+    system: cyclic;
+    symbols: "\\1F44D";
+    suffix: " ";
+}
+@counter-style
+identifier
+{
+system
+:
+cyclic
+;
+symbols
+:
+"\\1F44D"
+;
+suffix
+:
+" "
+;
+}
+
+@counter-style
+
+identifier
+
+{
+
+system
+
+:
+
+cyclic
+
+;
+
+symbols
+
+
+:
+
+"\\1F44D"
+
+;
+
+suffix
+
+:
+
+" "
+
+;
+
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+@counter-style identifier {
+  system: cyclic;
+  symbols: "\\1F44D";
+  suffix: " ";
+}
+
+@counter-style identifier {
+  system: cyclic;
+
+  symbols: "\\1F44D";
+
+  suffix: " ";
+}
+
+`;
+
+exports[`custom-selector.css 1`] = `
+@custom-selector :--heading h1, h2, h3, h4, h5, h6;
+@custom-selector :--heading h1,h2,h3,h4,h5,h6;
+@custom-selector  :--heading  h1  ,  h2  ,  h3  ,  h4  ,  h5  ,  h6  ;
+@custom-selector :--heading
+    h1, h2, h3, h4, h5, h6;
+@custom-selector
+    :--heading
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6;
+@custom-selector
+:--heading
+h1
+,
+h2
+,
+h3
+,
+h4
+,
+h5
+,
+h6
+;
+@custom-selector
+
+:--heading
+
+h1
+
+,
+
+h2
+
+,
+
+h3
+
+,
+
+h4
+
+,
+
+h5
+
+,
+
+h6
+
+;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+@custom-selector :--heading h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6;
+
+`;
+
+exports[`debug.css 1`] = `
+@debug 10em + 12em;
+@debug 10em+12em;
+@debug  10em  +  12em  ;
+@debug 10em
+    +
+    12em
+    ;
+@debug
+10em
++
+12em
+;
+@debug
+
+10em
+
++
+
+12em
+
+;
+@debug $very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-var + $very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-var;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@debug 10em + 12em;
+@debug 10em+12em;
```

Comment:
```scss
@debug 10em + 12em;
```

Or would that be too difficult to support?

Reply:
@j-f1 no, problems in https://github.com/prettier/prettier/blob/master/src/language-css/printer-postcss.js#L341, when i use two space or newlines all look good, seems it is separate issue, i do PR and when rebase